src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int firstProperty;
+    private int secondProperties;
+
+    public IsoscelesTrapezoid(String color) {
+        super(color);
+    }
+
+    public IsoscelesTrapezoid(String color, int firstProperty, int secondProperties) {
+        super(color);
+        this.firstProperty = firstProperty;
+        this.secondProperties = secondProperties;
+    }
+
+    @Override
+    public int getArea() {
+        return (firstProperty + secondProperties) / 2;
```

Comment:
It seems to me that the area of an isosceles trapezoid is calculated using a different formula